### PR TITLE
Add new figure for regions and rewrite text to use it.

### PR DESCRIPTION
**Pull request by TallJimbo**
As promised; sorry this took me so long.  In addition to the new figure, I also reordered the regions section of the paper quite a bit to use the figure as an organizational aid, and added a bit of text to highlight how the LSST and Astropy libraries so far are very complementary if we can find a way to integrate them.

### CODE REVIEW

File: SPIE2016-9913-16/9913-16.tex [hunk 1]
```diff
@@ -169,36 +169,53 @@ \subsection{Region handling}
...
-\caption{Four distinct types of region. This paper discusses all except discrete spherical.}
+\begin{center}
+\includegraphics[width=0.5\textwidth]{regions}
+\end{center}
+\caption{Four ways of representing regions, and the approximate domains of the LSST (blue) and Astropy (green) packages for manipulating them.}
 \label{fig:regions}
 \end{figure}
 
-Astropy has no immediate plans for a discretized efficient representation of a pixel region, and this is something that LSST can contribute to the Astropy region handling package.
+\subsection{Region handling}
```
Comment (timj) [inline SPIE2016-9913-16/9913-16.tex]: This text now has two "Region Handling" subsections. I am assuming you meant to remove the earlier text?

Comment (TallJimbo) [inline SPIE2016-9913-16/9913-16.tex]: Yes, sorry.  I actually saw this as a rebase conflict, and thought I'd fixed it; I'm guessing I just forgot to save before doing "git add".

Comment (timj) [inline SPIE2016-9913-16/9913-16.tex]: Sorry. I tweaked one line in the section by adding the URL. I had forgotten you were editing the same bit. I'll fix it up if you like.

Comment (TallJimbo) [inline SPIE2016-9913-16/9913-16.tex]: Please go ahead.

**Pull request merged**